go/vt/vterrors/vterrors.go

```diff
@@ -9,25 +9,18 @@ package vterrors
 import (
 	"fmt"
 
-	mproto "github.com/youtube/vitess/go/mysql/proto"
-	pb "github.com/youtube/vitess/go/vt/proto/vtrpc"
-)
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/codes"
 
-const (
-	// TabletError is the base VtTablet error. All VtTablet errors should be 4 digits, starting with 1.
-	TabletError = 1000
-	// UnknownTabletError is the code for an unknown error that came from VtTablet.
-	UnknownTabletError = 1999
-	// VtgateError is the base VTGate error code. All VTGate errors should be 4 digits, starting with 2.
-	VtgateError = 2000
-	// UnknownVtgateError is the code for an unknown error that came from VTGate.
-	UnknownVtgateError = 2999
+	mproto "github.com/youtube/vitess/go/mysql/proto"
+	"github.com/youtube/vitess/go/vt/proto/vtrpc"
+	"github.com/youtube/vitess/go/vt/tabletserver"
```

Comment:
I don't actually want to pull tabletserver in as a dependency here... I'll probably need to split tablet_error out as a different package.